src/views/dashboard/Pam/RiskSummary.vue

```diff
@@ -145,7 +153,7 @@ export default {
         series: [
           {
             type: 'bar',
-            data: data.map(item => item.value),
+            data: filteredData.map(item => item.value),
             barWidth: '60%',
             label: {
               show: false,
```

Comment:
There doesn't seem to be any known issue with the code provided. However, it might help if there were some more specific details about what you're looking for in terms of performance or functionality improvements.